### DISCUSSION TITLE
Adonis 5: Handler custom 404

### DISCUSSION THREAD

**Discussion by Nikolastual** [adonisjs/core#1473] [Help]
Hi!

With the base exception handler, I can't send a 404 page.

```
import Logger from '@ioc:Adonis/Core/Logger'
import HttpExceptionHandler from '@ioc:Adonis/Core/HttpExceptionHandler'

export default class ExceptionHandler extends HttpExceptionHandler {
	protected statusPages = {
		'404': 'errors.not-found',
		'500..599': 'errors.server-error',
	}

	constructor () {
		super(Logger)
	}
}
```

The exception returns: E_ROUTE_NOT_FOUND: Cannot GET: / test
it should return: errors.not-found

Thanks

**Comment by thetutlage** [comment]
From the docs

<img src="https://user-images.githubusercontent.com/1706381/90338727-45499880-e009-11ea-8eba-604bb16085a2.jpg" width="200px" />

**Reply by Nikolastual** [reply]
Even with:
`protected disableStatusPagesInDevelopment = false`
I still have: E_ROUTE_NOT_FOUND: Cannot GET: /test

it works with 500 errors but not with 404

**Comment by KararTY** [comment]
Also having this issue with NODE_ENV set to `production`.

On 404, I get `E_ROUTE_NOT_FOUND` instead of the edge view rendering.

On 50X, I get the edge view render as intended.

**Comment by thetutlage** [comment]
Can you please file it as an issue on the `@adonisjs/http-server` repo and I will look into it

**Comment by scottie** [comment]
Can confirm same issue as above when in production.
https://github.com/adonisjs/http-server/issues/60

**Reply by KararTY** [reply]
What version of adonisjs/core (Or adonisjs/http-server) are you on?

This works for me after https://github.com/adonisjs/http-server/issues/22, and it still works on adonisjs/http-server 5.8.0.

**Haven't yet tested on 5.9.0.**

Project: https://github.com/kararty/befriendlier-web
